### ISSUE TITLE
Add/Test MozJPEG in a production image pipeline

### ISSUE THREAD

**Issue by joemaller** [ideasonpurpose/tooling-starter-pack#18]
https://github.com/mozilla/mozjpeg

https://github.com/imagemin/imagemin-mozjpeg

**Comment by Wowfunhappy** [comment]
All of the below needs to happen before we can fully move to MozJPEG for image compression.

- [ ]  **Create simplified build system for image compression**, which a developer can safely "ignore" if he or she so chooses. PMs will run this system before staging a website for client review, but the site must be able to appear correctly without it. Rely on relative paths to avoid modifying code between src and dist.
- [ ] **Update our [developer guidelines](https://github.com/ideasonpurpose/developer-guidelines)** to indicate that developers should not be compressing images themselves outside of a toolchain. If they don't use a toolchain, they should not be alarmed by large image file sizes.
- [ ] **Change guidance for designers**. Designers should _always_ export JPEGs at full quality (100%), which is the exact opposite of what we've said previously.

(Joe, feel free to split this into separate tickets. However, I think all three tasks have to happen at once.)

**Comment by joemaller** [comment]
@Wowfunhappy Another thing to consider is encouraging developers who haven't used the `src` and `dist` directory structure to adopt that project layout. There will also be the assorted collection of modern webdev dotfiles and cruft which seems messy at first.

**Comment by joemaller** [comment]
Closed in 48f8a49cdba6b6a9c53d7292fecb3d5266658bdf. Keeping this open for the documentation tasks.
